include/util/coordinate_calculation.hpp

```diff
@@ -85,16 +85,6 @@ double bearing(const Coordinate first_coordinate, const Coordinate second_coordi
 // Get angle of line segment (A,C)->(C,B)
 double computeAngle(const Coordinate first, const Coordinate second, const Coordinate third);
 
-// find the center of a circle through three coordinates
```

Comment:
Well. It is something I currently work on and I feel code gets lost in the depths of git if once removed. After all you have no further documentation on it, unless you keep an issue or something around that documents that this code was there and might be needed again.

Since it is most likely needed in my other PR or at least could be used to compute the turn radius for turn penalty functions, I'd rather not forget that we have it.